src/AdminPanel/modules/LoginPage/LoginPage.tsx

```diff
@@ -1 +1,41 @@
-export const LoginPage = () => <div data-cy="LoginPageDiv">Login page</div>;
+import { useForm, SubmitHandler } from 'react-hook-form';
+
+type LoginData = {
+  username: string;
+  password: string;
+};
+
+export const LoginPage = () => {
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+    reset,
+  } = useForm<LoginData>({ mode: 'onTouched', defaultValues: { username: '', password: '' } });
+
+  const onSubmit: SubmitHandler<LoginData> = data => {
+    reset();
+    alert(JSON.stringify(data));
+  };
+
+  return (
+    <form data-cy="admin-login-form" onSubmit={handleSubmit(onSubmit)}>
+      <label data-cy="username-label" htmlFor="username">
+        Username:
+      </label>
+      <input data-cy="username-input" {...register('username', { required: 'Username is required!' })} />
```

Comment:
please move form fields to their own file

Reply:
resolved